src/schema/v2/homeView/sections/ExploreByCategory.ts

```diff
@@ -0,0 +1,63 @@
+import { ContextModule } from "@artsy/cohesion"
+import { HomeViewSection } from "."
+import { HomeViewSectionTypeNames } from "../HomeViewSection"
+import { URL, URLSearchParams } from "url"
+import { connectionFromArray } from "graphql-relay"
+
+const marketingColletionCategories = {
+  Medium: {
+    id: "Medium",
+    title: "Medium",
+  },
+  Movement: {
+    id: "Movement",
+    title: "Movement",
+  },
+  "Collect by Color": {
+    id: "Collect by Color",
+    title: "Color",
+  },
```

Comment:
So far I'm thinking that the Eigen component can prefer `href` if it's provided and otherwise fallback to `entityType` + `entityID` values to determine how to navigate internally.

Reply:
`id` might mislead here, `id` generally represents the resource ID, I'd rather call it something different

Reply:
We're still using "Collect by Color" (and others) as an identifier though. Very similar to how we reference [attribution classes](https://github.com/artsy/metaphysics/blob/a9f38eab4651cba47ba7fb5deeacc0ef67ad2fcf/src/lib/attributionClasses.ts#L4) and [artwork medium types](https://github.com/artsy/metaphysics/blob/0416dcd2f1a2309bb318f90d546bd13a23f9ebd2/src/lib/artworkMediums.ts), both of which lack a 1-1 mapping with a backend resource type.

I'm open to suggestions but I think using `id` would be consistent. I consider it broadly fits as an identifier but not specifically an internal database record identifier.